Tasks/AzureDevOpsTestPlanV0/automatedTestInvoker.ts

```diff
@@ -0,0 +1,28 @@
+import * as tl from 'azure-pipelines-task-lib/task'
+import { executepythontests } from './Invokers/pythonivoker'
+import { executemaventests } from './Invokers/maveninvoker'
+import { executegradletests } from './Invokers/gradleinvoker'
+
+export async function testInvoker(testsToBeExecuted: string[]) {
+
+    const testLanguage = tl.getInput('testLanguageInput');
+
+    if (testLanguage !== null && testLanguage !== undefined) {
+
+        if (testLanguage.includes("Java-Maven")) {
```

Comment:
Suggestion: Should we implement a composition relationship here. Where there is an interface with a method executeTests and each language invoker would be implementing that interface. This would make onboarding of new languages cleaner because it adds in the abstraction.

Something similar to how we have Parser.cs and there is an interface ITestResultParser which is implemented by all trxparser, junit etc.

Reply:
good suggestion, please let me know if I can take it up in code refactoring.?

Reply:
Yeah, I am okay taking it up as part of refactoring.